libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -48,4 +61,9 @@ export class BookSearchComponent {
       this.store.dispatch(clearSearch());
     }
   }
+
```

Comment:
searchExample() {
    this.searchForm.controls.term.setValue('javascript');
    this.searchBooks();
  }
  
  in above method in this file, do you need  `this.searchBooks();` statement now as we are going with instant search.